packages/mainsail/source/transaction.service.ts

```diff
@@ -389,34 +398,36 @@ export class TransactionService extends Services.AbstractTransactionService {
 			await this.#ledgerService.disconnect();
 		}
 
+		let signedTransactionBuilder;
+
 		if (input.signatory.actsWithMnemonic()) {
-			transaction.sign(input.signatory.signingKey());
+			signedTransactionBuilder = await transaction.sign(input.signatory.signingKey());
 		}
 
 		if (input.signatory.actsWithConfirmationMnemonic()) {
-			transaction.sign(input.signatory.signingKey());
-			transaction.secondSign(input.signatory.confirmKey());
+			signedTransactionBuilder = await transaction.sign(input.signatory.signingKey());
+			// transaction.secondSign(input.signatory.confirmKey());
```

Comment:
we will need to fix `secondSign*` stuff later, atm we don't have these methods exposed